app.py

```diff
@@ -13,30 +23,33 @@ def collect_images(directory):
                 image_paths.append(os.path.join(root, file))
     return image_paths
 
-def getListImage(image_paths):
+# Function to get image data list from image paths
+def get_list_image(image_paths):
     request = []
     for image_data in image_paths:
-        print(image_data)
         with open(image_data, 'rb') as f:
             image = f.read()
             request.append(Part.from_data(data=image, mime_type="image/jpeg"))
-            
     return request
 
-def addText(request, number, baseQuestions=None):
+# Function to add text to the request
+def add_text(request, baseQuestions=None):
     if baseQuestions == None:
```

Comment:
```suggestion
def add_text(request, base_question=None):
```

If you apply the suggestion be careful to change the variable name everywhere else in the code